samples/weather_function/Dockerfile

```diff
@@ -0,0 +1,38 @@
+FROM openjdk:8u171-jdk-stretch as builder
+WORKDIR /opt
+ENV KOTLIN_NATIVE_VER "0.7.1"
+RUN wget "https://github.com/JetBrains/kotlin-native/releases/download/v0.7.1/kotlin-native-linux-$KOTLIN_NATIVE_VER.tar.gz"
```

Comment:
These lines should all be one instruction to enable caching, this is an inefficient usage of a Dockerfile

```
RUN wget ... \
  && tar .... \
  && apt-get 
```

For your `apt-get` you need to specify `-qy` as an argument to suppress unexpected prompting behaviour.